themes/atomic.omp.json

```diff
@@ -0,0 +1,326 @@
+{
+  "$schema": "https://raw.githubusercontent.com/JanDeDobbeleer/oh-my-posh/main/themes/schema.json",
+  "final_space": false,
+  "osc99": true,
+  "blocks": [
+    {
+      "type": "prompt",
+      "alignment": "left",
+      "segments": [
+        {
+          "type": "shell",
+          "style": "diamond",
+          "leading_diamond": "╭─",
+          "trailing_diamond": "",
+          "foreground": "#ffffff",
+          "background": "#0077c2",
+          "properties": {
+            "prefix": " "
+          }
+        },
+        {
+          "type": "root",
+          "style": "diamond",
+          "leading_diamond": "",
+          "trailing_diamond": "",
+          "foreground": "#FFFB38",
+          "background": "#ef5350",
+          "properties": {
+            "root_icon": "\uf292",
+            "prefix": "<parentBackground>\uE0B0</> "
+          }
+        },
+        {
+          "type": "path",
+          "style": "powerline",
+          "powerline_symbol": "\uE0B0",
+          "foreground": "#2d3436",
+          "background": "#FF9248",
+          "properties": {
+            "home_icon": "",
+            "folder_icon": "\uf413 ",
+            "style": "folder",
+            "prefix": "<parentBackground>\uE0B0</> \ufc6e\uf553",
```

Comment:
why do you need the <parentBackground>\uE0B0</> here?

Reply:
That was going to be remark indeed, something is off there (looks pretty col other than that).

Reply:
It's correct, just to style the theme and make it cool, I really left it that way because it seems quite elegant to me.

In this case, is it wrong that I put this?